examples/outliers/alibi-detect-combiner/notebook.ipynb

```diff
@@ -0,0 +1,1382 @@
+{
```

Comment:
<p>There's still a note about making a diagram, not sure if we still want to add this diagram for this PR. It may be worth adding one with Draw.IO if there's time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1306/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I can see that there are already other diagrams, so just removing this TODO should be enough

Reply:
Removing.